tests/Integration/Behaviour/Features/Context/CurrencyFeatureContext.php

```diff
@@ -63,12 +66,43 @@ public function cleanCurrencyFixtures()
         $this->currencies = [];
     }
 
+    /**
+     * @Given currency :reference with isoCode :isoCode exists
+     */
+    public function createCurrencyWithIsoCode($reference, $isoCode)
+    {
+        //Currency::getIdByIsoCode only returns not deleted currency so we check the storage to avoid duplicate contents
+        if (SharedStorage::getStorage()->exists($reference)) {
+            /** @var Currency $currency */
+            $currency = SharedStorage::getStorage()->get($reference);
+            if ($currency->iso_code == $isoCode) {
+                return;
```

Comment:
Add a comment to tell the iso currency is already defined.

Reply:
Added in the previous comment